src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 20;
+  const subsequentDiscount = 50;
+  const basicDays = 3;
+  const subsequentDays = 7;
+  const rentPrice = 40;
+  let carRentalAmount = days * rentPrice;
+
+  if (days >= basicDays && days < subsequentDays) {
+    carRentalAmount -= basicDiscount;
```

Comment:
you actually can just return it right away and make carRentalAmount as a const
```suggestion
    return carRentalAmount - basicDiscount;
```

Reply:
using const instead of let is better for performance

Reply:
Thanks! Done :)